src/main/java/org/parabot/core/parsers/servers/LocalServers.java

```diff
@@ -86,17 +86,24 @@ public void execute() {
                 if ((bank = object.get("bank")) != null) {
                     bankTabs = (int) bank;
                 }
+                String     uuidStr = (String) object.get("uuid"); // optional
```

Comment:
What if no uuid is defined in the json

Reply:
In which case it is ignored. Going to change default value from 0 to -1 in next commit.